test/unit/flux-capacitor.ts

```diff
@@ -279,6 +280,34 @@ suite('FluxCapacitor', ({ expect, spy, stub }) => {
         expect(dispatch).to.be.calledWithExactly(action);
       });
     });
+
+    describe('awaitBiasingRehydration()', () => {
+      it('should dispatch given action if data loaded from browser', () => {
+        const action = 'action';
+        const getState = store.getState = spy();
+        const dispatch = store.dispatch = spy();
+        stub(Selectors, 'realTimeBiasesHydrated').returns(true);
+
+        flux.awaitBiasingRehydration(<any>action);
+        expect(getState).to.be.calledWith();
+        expect(dispatch).to.be.calledWith(action);
+      });
+
+      it('should wait on personalization biasing rehydrated if data not loaded from browser', () => {
+        const action = 'action';
+        const realtimeBiasesHydrated = stub(Selectors, 'realTimeBiasesHydrated').returns(false);
+        const once = flux.once = spy();
+        const dispatch = store.dispatch = spy();
+        store.getState = spy();
+
+        flux.awaitBiasingRehydration(<any>action);
+        expect(dispatch).to.not.be.called;
+        expect(once).to.be.calledWith(Events.PERSONALIZATION_BIASING_REHYDRATED);
+
+        once.getCall(0).args[1]();
```

Comment:
rather than doing this, check the documentation for `sinon.match()` and look at how we use it elsewhere in the tests

Reply:
Are you referring to the calledWith? If so then wouldn't an exact argument rather than sinon.match be better?